hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/security/TestOzoneBlockTokenSecretManager.java

```diff
@@ -329,7 +329,7 @@ private X509Certificate generateExpiredCert(String dn,
       KeyPair pair, String algorithm) throws CertificateException,
       IllegalStateException, IOException, OperatorCreationException {
     Date from = new Date();
-    Date to = new Date(from.getTime() + 100L);
+    Date to = from;
     return generateTestCert(dn, pair, algorithm, from, to);
```

Comment:
you can directly pass `from` instead of `to` in `generateTestCert`
something like -  ```generateTestCert(dn, pair, algorithm, from, from);```
no need to have
``Date to = from;``

Reply:
Yes, but I think it may be more readable to keep the fields of "from" and "to"?

Reply:
For readability better to add a comment explaining why we are passing `from` for both arguments.
Though trivial assigning unnecessary would have cost and isn't a good practice IMO

Reply:
Got it. Have added a comment for readability.